production/scrapers/minnesota_vaccine.R

```diff
@@ -0,0 +1,90 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+minnesota_vaccine_pull <- function(x){
+    "1VhAAbzipvheVRG0UWKMLT6mCVQRMdV98lUUkk-PCYtQ" %>%
+        googlesheets4::read_sheet(sheet = "Minnesota Vaccine")
+}
+
+minnesota_vaccine_restruct <- function(x){
+    x %>%
+        mutate(Date = lubridate::ymd(`Date`)) %>%
+        filter(!is.na(Date)) %>% 
+        filter(Date == max(Date))
+}
+
+minnesota_vaccine_extract <- function(x, exp_date = Sys.Date()){
+    
+    error_on_date(first(x$Date), exp_date)
+    
+    check_names(x, c(
+        "Date", 
+        "Facility", 
+        "Staff Received 1st Dose", 
+        "Staff Received 2nd Dose", 
+        "Client Received 1st Dose", 
+        "Client Received 2nd Dose"
+        ))
+    
+    x %>%
+        select(
+            Name = `Facility`,
+            Residents.Initiated = `Client Received 1st Dose`,
+            Residents.Completed = `Client Received 2nd Dose`, 
+            Staff.Initiated = `Staff Received 1st Dose`, 
+            Staff.Completed = `Staff Received 2nd Dose`) %>% 
```

Comment:
Should we add in Vadmin = Initiated + Completed in here? Can't remember how we decided to approach this